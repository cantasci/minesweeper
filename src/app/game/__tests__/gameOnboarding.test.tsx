import React from 'react';

import { cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';

import GameOnboarding from '../gameOnboarding';

describe('<GameOnboarding />', () => {
  afterEach(() => {
    jest.clearAllMocks();
    cleanup();
  });
  it('renders correctly and to match snapshot', () => {
    const tree = renderer.create(<GameOnboarding />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
