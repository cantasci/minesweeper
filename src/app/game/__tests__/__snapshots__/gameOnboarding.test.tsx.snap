// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameOnboarding /> renders correctly and to match snapshot 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 css-f8nw7v-MuiTypography-root"
>
  Press START to play Minesweeper!
</p>
`;
